ansible_collections/arista/avd/roles/eos_designs/docs/how-to/wan.md

```diff
@@ -853,3 +853,14 @@ wan_virtual_topologies:
           - names: [internet]
             preference: alternate
 ```
+
+### WAN Validation
+
+`eos_validate_state` is being enriched to support new tests for WAN designs. The tests listed below are validating WAN designs.
+
+| AVD Test Class | ANTA Test Class | Description |
+| -------------- | --------------- | ----------- |
+| AvdTestInterfacesState | VerifyInterfacesStatus | Validate the DPS interface status. |
+| AvdTestBGP | VerifyBGPSpecificPeers | Validate the different BGP Address Families Identifiers (AFI) and Subsequent Address Family Identifiers (SAFI) sessions state. |
+| AvdTestIPSecurity | VerifySpecificIPSecConn | Validates the establishment of IP security connections for a peer within the default VRF. In its current state, the test validates only IPsec connections defined as static peers under the `router path-selection` section of the configuration. |
```

Comment:
we are talking about WAN here so lets reword - same for the STUN one, you can remove "for WAN scenarios" as we are talking about WAN here
```suggestion
| AvdTestIPSecurity | VerifySpecificIPSecConn | Validates the establishment of IP security connections for each static peer under the `router path-selection` section of the configuration. |
```

Reply:
fix